README.md

```diff
@@ -48,8 +50,83 @@ const { useInitZkStore, useZKStore, useGetLatestProof } =
             num: state.num + 1,
           };
         }),
-    })
+    }),
+    
+    // Define the state variable that you want to trigger a proof generation on
+    // NOTE Functions will never trigger a proof generation
```

Comment:
this is can be interpreted in a very wrong way. maybe something like "functions will still be omitted in the prover and verifier input, only variables are considered"

Reply:
I'll just say:
`// Define state properties that should trigger proof generation when mutated`. 
`// You can define assertions in the Actions so that the proofs are generated based only on VALID STATE TRANSITIONS`. 

No need to expand on the functions yet. That's just our internal way to implement some feature, not so much something that the user needs to immediately know.

Reply:
The thing is that the values in the arrays are inferred from the interface so the editor may also suggest and accept function names, this is a way of saying that those will be ignored by default